### PR TITLE
chore(ci): general maintenance

### PR DESCRIPTION
**Pull request by peterdeme**

### CODE REVIEW

**Comment by peterdeme** [comment]
Holy 💩 , we have a lots of linter errors, I'll need to fix it.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -2,33 +2,44 @@ name: build
...
+          extensions: ast, mbstring, intl
           ini-values: post_max_size=256M, max_execution_time=180
           tools: composer:v2
 
       - name: Deps
         run: composer install --no-interaction
 
       - name: Quality
-        if: matrix.php-versions == '7.4'
+        if: matrix.php-versions == '7.3'
```
Comment (ferhatelmas) [inline .github/workflows/ci.yml:39]: yes, this is better

File: README.md [hunk 2]
```diff
@@ -1,40 +1,58 @@
...
+    <a href="https://github.com/GetStream/stream-php/issues">Report Bug</a>
+    ·
+    <a href="https://github.com/GetStream/stream-php/issues">Request Feature</a>
+</p>
 
-Note that there is also a higher level [Laravel](https://github.com/getstream/stream-laravel) integration which hooks into the Eloquent ORM.
+## 📝 About Stream
 
-You can sign up for a Stream account at https://getstream.io/get_started.
+You can sign up for a Stream account at our [Get Started](https://getstream.io/chat/get_started/) page.
```
Comment (ferhatelmas) [inline README.md:23]: this is feed sdk, url is wrong

File: README.md [hunk 3]
```diff
@@ -1,40 +1,58 @@
...
 
-### Installation
+You can use this library to access chat API endpoints server-side.
```
Comment (ferhatelmas) [inline README.md:25]: same here

File: README.md [hunk 4]
```diff
@@ -1,40 +1,58 @@
...
 
-#### Composer
+For the client-side integrations (web and mobile) have a look at the JavaScript, iOS and Android SDK libraries ([docs](https://getstream.io/chat/)).
```
Comment (ferhatelmas) [inline README.md:27]: tweak your intro for feeds

File: README.md [hunk 5]
```diff
@@ -229,28 +247,20 @@ $client->users()->delete('42');
...
-
-```
-vendor/bin/phpunit --testsuite "Unit Test Suite"
-```
-
-### Copyright and License Information
+## ✍️ Contributing
 
-Project is licensed under the [BSD 3-Clause](LICENSE).
+We welcome code changes that improve this library or fix a problem, please make sure to follow all best practices and add tests if applicable before submitting a Pull Request on Github. We are very happy to merge your code in the official repository. Make sure to sign our [Contributor License Agreement (CLA)](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) first. See our [license file](./LICENSE) for more details.
```
Comment (ferhatelmas) [inline README.md:260]: feed sdks are BSD-3, don't change it

File: composer.json [hunk 6]
```diff
@@ -10,25 +14,28 @@
...
     "require": {
         "php": ">=7.3",
         "guzzlehttp/guzzle": "^6.3.3 || ^7.0.1",
         "firebase/php-jwt": "^v5.0.0"
     },
     "require-dev": {
-        "friendsofphp/php-cs-fixer": "^2.18.0",
-        "phan/phan": "^4.0.3",
-        "phpunit/phpunit": "^8.2.1",
-        "ramsey/uuid": "^3.8"
```
Comment (ferhatelmas) [inline composer.json:22]: I loved your refactor

File: tests/integration/ReactionTest.php [hunk 7]
```diff
@@ -62,17 +61,22 @@ protected function setUp():void
...
         $response = $this->user1->addActivity($activity_data);
         $this->activity_id = $response['id'];
         $this->reactions = $this->client->reactions();
     }
 
+    private function generateGuid()
+    {
+        return vsprintf('%s%s-%s-%s-%s-%s%s%s', str_split(bin2hex(random_bytes(16)), 4));
+    }
+
```
Comment (ferhatelmas) [inline tests/integration/ReactionTest.php:79]: might want to have a test util though

**Review by ferhatelmas — CHANGES_REQUESTED** [review]
* readme and license updates are wrong
* dropped dep is great but could use a helper to reduce repetition
* rest is good (this will enable us to drop legacy auth easily)

File: README.md (hunk 5 above)
Comment (peterdeme) [reply README.md:260]: copy-pasta 🍝

File: composer.json (hunk 6 above)
Comment (peterdeme) [reply composer.json:22]: 🙏

File: README.md (hunk 2 above)
Comment (peterdeme) [reply README.md:23]: fixed

File: README.md (hunk 3 above)
Comment (peterdeme) [reply README.md:25]: fixed

File: README.md (hunk 5 above)
Comment (peterdeme) [reply README.md:260]: added back

**Review by ferhatelmas — APPROVED** [review]
LGTM

**Review by ferhatelmas — APPROVED** [review]

**Pull request merged**